resources/user.rb

```diff
@@ -0,0 +1,106 @@
+property :username, String, name_attribute: true, desired_state: false, identity: true
+property :password, String, desired_state: false, identity: true, sensitive: true
+property :first_name, String, desired_state: false, identity: true, default: ''
+property :last_name, String, desired_state: false, identity: true, default: ''
+property :email, String, desired_state: false, identity: true, default: ''
+property :roles, Array,  desired_state: false, identity: true, default: []
+property :api_endpoint, String, desired_state: false, identity: true,
+                                default: lazy { node['nexus3']['api']['endpoint'] }
+property :api_username, String, desired_state: false, identity: true,
+                                default: lazy { node['nexus3']['api']['username'] }
+property :api_password, String, desired_state: false, identity: true, sensitive: true,
+                                default: lazy { node['nexus3']['api']['password'] }
+
+load_current_value do |desired|
+  api_endpoint desired.api_endpoint
+  api_username desired.api_username
+  api_password desired.api_password
```

Comment:
The 3 above lines are useless because these are `identity` properties

Reply:
removed